### PR TITLE
More mail sprites

### PR DESCRIPTION
**Pull request by Gannets**
## About the PR 
- Adds a new sprite for mail crates.
- Better "full" light for the QM mail-chute. Full doesn't mean "there's too much stuff in here, you can't add any more" it actually means "not empty."

image

## Changelog 

```changelog
(u)Gannets
(+)Added a new sprite for mail crates.
```

### CODE REVIEW

File: code/obj/storage/crates.dm [hunk 1]
```diff
@@ -493,6 +493,13 @@ TYPEINFO(/obj/storage/crate/chest)
 			bux.pixel_x = rand(-9, 9)
 			bux.pixel_y = rand(0, 6)
 
+/obj/storage/crate/mail
+	name = "internals crate"
+	desc = "An internals crate."
```
Comment (TheTyrant25) [inline code/obj/storage/crates.dm:498]: is this the correct name and desc? doesn't seem right.

**Review by TheTyrant25 — COMMENTED** [review]
looks really good

File: code/obj/storage/crates.dm (hunk 1 above)
Comment (Gannets) [reply code/obj/storage/crates.dm:498]: lol whoops it's not, but it gets set in shippingmarket.dm anyway

Comment (Gannets) [reply code/obj/storage/crates.dm:498]: will fix

**Pull request merged**